group-generators/generators/gitcoin-passport-holders/index.ts

```diff
@@ -7,45 +7,47 @@ import {
   GroupGenerator,
 } from "topics/group-generator";
 
-// Generated from factory.sismo.io
-
 const generator: GroupGenerator = {
-  generationFrequency: GenerationFrequency.Once,
+  generationFrequency: GenerationFrequency.Daily,
 
   generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
-    const restProvider = new dataProviders.RestProvider();
 
+    type Passport = {
+      address: string,
+      score: string,
+      status: string,
+      last_score_timestamp: string,
+      evidence: {
+        type: string,
+        success: boolean,
+        rawScore: string,
+        threshold: string
+      },
+      error: any
+    };
+
+    const restProvider = new dataProviders.RestProvider();
     const gitcoinPassportHolders: FetchedData = {};
-    let res: any;
-    const url = "https://api.scorer.gitcoin.co";
+    const url = "https://indexer-grants-stack.gitcoin.co/data/passport_scores.json";
 
     const apiConfig: ApiConfig = {
-      url: url + "/analytics/score/13?limit=1000",
+      url: url,
       method: "GET",
       headers: {
-        authorization: `Bearer ${process.env.GITCOIN_API_KEY}`,
         accept: "application/json",
       },
     };
 
-    res = await restProvider.fetchData(apiConfig);
+    const res: any = await restProvider.fetchData(apiConfig);
 
-    res.items.forEach((user: any) => {
-      gitcoinPassportHolders[user.address] = 1;
+    res.forEach((passport: Passport) => {
+      if(passport.status == "DONE") {
```

Comment:
I will remove the filter here. 
this seems to be used for "DONE" or "PROCESSING" like an internal process from gitcoin passport to refresh the value.

Reply:
But for all the addresses that have the PROCESSING status here's their Data:
```
  {
    "address": "0x461337d4f089adf16455acb785415f6437da0c24",
    "score": null,
    "status": "PROCESSING",
    "last_score_timestamp": null,
    "evidence": null,
    "error": null
  },
```
So we can't put their score as a value because it's null.

What I propose to do is (considering that a Gitcoin passport cannot be deleted), we fetch the latest group and we do a union we the new fetched ones so we can keep the score of the address even if they have a PROCESSING status.
What do you think?